### ISSUE TITLE
Can this be installed as a Homebridge plugin?

### ISSUE THREAD

**Issue by jaggles** [jfdoube/homebridge-sinope#2]
Can this be installed as a plugin for Homebridge using "npm install -g homebridge-sinope"? I get an error when I try that.

**Comment by jfdoube** [comment]
it's not in the npm repo yet. Just trying to find a bit of time to fix some logged bugs and create an npm package.

**Comment by jaggles** [comment]
Is there a way for me to manually install it now, to test it out?

**Comment by rjawl** [comment]
@jaggles @jfdoube Yeah I'd love to give this a try myself.

**Comment by jfdoube** [comment]
Haven't had the time to look into it. I should have some spare time soon.

**Comment by yungeconomist** [comment]
Would be very interested to help and test this integration as it develops!

**Comment by jfdoube** [comment]
Absolutely, feel free to fork the repo and send PR, as i don't currently have much time to evolve the plugin.

**Comment by radavidescu** [comment]
A friendly user on reddit provided [instructions on installing this plugin manually](https://www.reddit.com/r/HomeKit/comments/61as76/homekit_electric_underfloor_heating/dfrna57/). here are the instructions in case it helps anyone like it helped me:

> basically, you just need to clone the repo on your computer/server, and then specify where the local file is on your machine while using the npm command.
> Clone the repo
> ` git clone https://github.com/jfdoube/homebridge-sinope.git`
> and install via npm
> ` sudo npm install -g ./homebridge-sinope`
> http://stackoverflow.com/questions/5786433/how-to-install-a-node-js-module-without-using-npm
